### ISSUE TITLE
Run-time error when clicking on the interpreter in the status bar (wxwidgets 3.0.2 on Linux)

### ISSUE THREAD

**Issue by Mailaender** [pkulchenko/ZeroBraneStudio#671]
```
Lua: Error while running chunk
src/editor/package.lua:149: wxLua: Unable to call an unknown method 'GetFirst' on a 'wxMenuItemList' type.
stack traceback:
	[C]: ?
	src/editor/package.lua:149: in function 'CloneMenu'
	src/editor/gui.lua:69: in function <src/editor/gui.lua:63>
	[C]: in function '?'
	src/main.lua:767: in main chunk
	[C]: ?
```

Crashes the application (version 1.30) when hitting either <kbd>Cancel</kbd>. Does nothing when clicking <kbd>OK</kbd>

**Comment by pkulchenko** [comment]
@Mailaender, are you running the version from the repository or the version you compiled? What wxwidgets/wxlua version are you using?

Could you check on the version that comes in the repository as I haven't been able to reproduce this issue?

If you are using your own version, then I'm not sure why it's missing `GetFirst` method as it's been available since 2.9 and is present in wxlua descriptions ([GetMenuItems](http://docs.wxwidgets.org/trunk/classwx_menu.html#abe0e1a4395a253e6fe45c821d7a2cf2b)).

**Comment by Mailaender** [comment]
![image](https://cloud.githubusercontent.com/assets/756669/17064637/c5c1324e-503d-11e6-9199-d7fdc622611e.png)

are the versions from my self-compiled version. The wxLua is actually a snapshot from SVN (latest trunk). I can't reproduce this with your release compiled against wxWidgets 2.9.5, but I get other glitches together with Gtk3 such as buttons having no outline, menu items vanishing on hover etc.

**Comment by pkulchenko** [comment]
> I get other glitches together with Gtk3 there such as buttons having no outline, menu items vanishing on hover etc.

Yes, those are known issues with the current version; can you also check running the code from wxwidgets31x-upgrade branch as it should address those deficiencies?

I'm not sure what to do with your issue as I'd expect this to work. The only suggestion I have is to try with wxwidgets 3.1 as this was the last released version.

If that's not an option for you, I can probably add a check for `GetFirst` method not being available, in which case the user will simply not get the menu popup, but will be able to continue without issues (and can still switch interpreters using the global menu).

**Comment by Mailaender** [comment]
I can confirm that this is fixed on your wxWidgets 3.1.1 branch. :white_check_mark:

**Comment by Mailaender** [comment]
Probably some breakage SUSE or me introduced during packaging so don't worry about it. It also isn't really a showstopper. Switching via the menu doesn't crash.
